SYCL/FilterSelector/filter_list_cpu_gpu_acc.cpp

```diff
@@ -12,16 +12,16 @@
 
 // RUN: env SYCL_DEVICE_FILTER="*" %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-GPU,CHECK-ACC,CHECK-HOST
 // RUN: env SYCL_DEVICE_FILTER=opencl,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-GPU,CHECK-ACC,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=cpu,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-GPU-NOT,CHECK-ACC-NOT,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=acc,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU-NOT,CHECK-GPU-NOT,CHECK-ACC,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=host %t.out | FileCheck %s --check-prefixes=CHECK-CPU-NOT,CHECK-GPU-NOT,CHECK-ACC-NOT,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=gpu,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU-NOT,CHECK-GPU,CHECK-ACC-NOT,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=cpu,acc,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-GPU-NOT,CHECK-ACC,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=cpu:0,acc:1,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-GPU-NOT,CHECK-ACC-NOT,CHECK-HOST
-// RUN: env CL_CONFIG_CPU_EMULATE_DEVICES=2 SYCL_DEVICE_FILTER=cpu:0,acc:1,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-GPU-NOT,CHECK-ACC,CHECK-HOST
-// RUN: env SYCL_DEVICE_FILTER=cuda:cpu,opencl:gpu:0,level_zero:acc:1,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU-NOT,CHECK-GPU-NOT,CHECK-ACC-NOT,CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=cpu,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=acc,host %t.out | FileCheck %s --check-prefixes=CHECK-ACC,CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=host %t.out | FileCheck %s --check-prefixes=CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=gpu,host %t.out | FileCheck %s --check-prefixes=CHECK-GPU,CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=cpu,acc,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-ACC,CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=cpu,acc,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-HOST
+// RUN: env CL_CONFIG_CPU_EMULATE_DEVICES=2 SYCL_DEVICE_FILTER=cpu,acc,host %t.out | FileCheck %s --check-prefixes=CHECK-CPU,CHECK-ACC,CHECK-HOST
+// RUN: env SYCL_DEVICE_FILTER=cuda:cpu,opencl:gpu,level_zero:acc,host %t.out | FileCheck %s --check-prefixes=CHECK-GPU,CHECK-HOST
 //
 // CHECK-ACC: Device: acc
-// CHECK-CPU: Device: cpu
 // CHECK-GPU: Device: gpu
+// CHECK-CPU: Device: cpu
```

Comment:
Why this line was moved? It does not look like intended change,

Reply:
I did refactoring of get_devices(), and it returns CPU devices after GPU devices.